freud/test_Hexatic.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2019 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+
+import freud
+from ovito.data import *
+
+def sense_check(k):
+    if k==0: print('WARNING! OVERWRITE k VALUE IN SCRIPT')
+    return
+
+def modify(frame, input, output):
+    #   USER SHOULD SET THIS MANUALLY BEFORE RUNNING SCRIPT
+    k = 6
```

Comment:
We should have all of these scripts default to giving parameters that will fail the check (i.e. set k=0 here). Since this isn't in the guts of some package I'm less interested in providing sensible defaults and more interested in avoiding having users shoot themselves in the foot. Also since in almost all cases there aren't good default choices, I'd prefer to be consistent and not provide any defaults rather than have a few exceptional cases where defaults are provided.

Reply:
Yes, that's the plan. Looks like that wasn't the case in this first commit, but will be in final PR.